ingest/mappings/dataset.json

```diff
@@ -0,0 +1,223 @@
+          "uri":{
+            "type":"string",
+            "index": "not_analyzed"
+          }
+        }
+      },
+      "dcoCommunities": {
+        "properties": {
+          "name":{
+            "type":"multi_field",
+            "fields":{
+              "name":{
+                "type":"string",
+                "index":"analyzed"
+              },
+              "exact":{
+                "type":"string",
+                "index":"not_analyzed"
+              }
+            }
+          },
+          "uri":{
+            "type":"string",
+            "index": "not_analyzed"
+          }
+        }
+      },
+      "portalGroups": {
+        "properties": {
+          "name":{
+            "type":"multi_field",
+            "fields":{
+              "name":{
+                "type":"string",
+                "index":"analyzed"
+              },
+              "exact":{
+                "type":"string",
+                "index":"not_analyzed"
+              }
+            }
+          },
+          "uri":{
+            "type":"string",
+            "index": "not_analyzed"
+          }
+        }
+      },
+      "projects": {
+        "properties": {
+          "name":{
+            "type":"multi_field",
+            "fields":{
+              "name":{
+                "type":"string",
+                "index":"analyzed"
+              },
+              "exact":{
+                "type":"string",
+                "index":"not_analyzed"
+              }
+            }
+          },
+          "uri":{
+            "type":"string",
+            "index": "not_analyzed"
+          }
+        }
+      },
+      "citations": {
+        "properties": {
+          "name": {
+            "type": "string",
+            "index": "not_analyzed"
+          },
+          "uri": {
+            "type": "string",
+            "index": "not_analyzed"
+          }
+        }
+      },
+      "distributions": {
+        "properties": {
+          "accessURL": {
+            "type": "string"
+          },
+          "files": {
+            "properties": {
+              "downloadURL": {
+                "type": "string"
+              },
+              "name": {
+                "type": "string"
+              },
+              "uri": {
+                "type": "string"
+              }
+            }
+          },
+          "name": {
+            "type": "string"
+          },
+          "uri": {
+            "type": "string"
+          }
+        }
+      },
+      "dcoId": {
+        "type": "string",
+        "index":"not_analyzed"
+      },
+      "doi": {
+        "type": "string",
+        "index":"not_analyzed"
+      },
+      "isDcoPublication": {
+        "type": "boolean"
+      },
+      "mostSpecificType": {
+        "type": "string",
+        "index":"not_analyzed"
+      },
+      "publicationYear": {
+        "type": "string"
+      },
+      "title": {
+        "type": "multi_field",
+          "fields": {
+            "name": {
```

Comment:
change "name" here to "title".  The inner field name should match the outer field name.  The two inner fields should be "title" and "exact" in this case.